keps/sig-node/compute-device-assignment.md

```diff
@@ -87,14 +94,17 @@ message PodResources {
 message ContainerResources {
     string name = 1;
     repeated ContainerDevices devices = 2;
+    string cpu_ids = 3;
 }
 
 // ContainerDevices contains information about the devices assigned to a container
 message ContainerDevices {
     string resource_name = 1;
     repeated string device_ids = 2;
+    int64 numaid = 3;
 }
 ```
+Cpu_ids is a list of cpu id in [cpuset format](https://man7.org/linux/man-pages/man7/cpuset.7.html#FORMATS). Each cpu id is a thread id or a core id in term of cadvisor.
```

Comment:
Using `cpuset` as a format (and especially in the string form) is really not the best idea in terms of engineering of APIs. It means that we are loosing validation on the grpc/protobuf level, and in additional all clients that supposed to consume that field will be required to implement parser for `cpuset` format (which is also non-trivial in terms of syntax and validation).
Having `repeated uint` allows to properly validate content and handle situations with empty/undefined conditions of this field cheaply.

Reply:
ok, no problem, cpuset format is mostly for human readability (it's not about unix domain socket). I just was focusing on golang/python clients (where cpuset parsing is publicly  available),

Reply:
To best of my knowledge, neither Python nor Go std libraries have `cpuset` string format parser available.
Passing string puts additional technical debt on the clients, either by implementing parser or by pulling non-std library.

Reply:
do we even need to put the cpusets of containers which are using the shared pool?  Should we just have a global default cpu set, and omit all pods using shared CPUs?  Or just omit containers using the shared pool all-together?

I agree that concrete lists of integers is better than cpuset notation.